enginetest/memory_engine_test.go

```diff
@@ -204,30 +204,72 @@ func TestSingleQueryPrepared(t *testing.T) {
 
 // Convenience test for debugging a single query. Unskip and set to the desired query.
 func TestSingleScript(t *testing.T) {
-	t.Skip()
```

Comment:
one nit of mine that others disagree with is that i usually reset this file back to main before commiting. 1) We forget to re skip this sometimes, which is bad because this suite shouldn't be tested. And b) for most GMS PRs its kind of annoying to review a PR where this file's testing changes outweigh the relevant diff.

Reply:
Agreed, will revert all memory_engine_test.go changes.